src/Sylius/Bundle/ThemeBundle/Asset/Package/PathPackage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Asset\Package;
+
+use Sylius\Bundle\ThemeBundle\Asset\PathResolverInterface;
+use Sylius\Bundle\ThemeBundle\Context\ThemeContextInterface;
+use Symfony\Component\Asset\Context\ContextInterface;
+use Symfony\Component\Asset\PathPackage as BasePathPackage;
+use Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class PathPackage extends BasePathPackage
+{
+    /**
+     * @var ThemeContextInterface
+     */
+    protected $themeContext;
+
+    /**
+     * @var PathResolverInterface
+     */
+    protected $pathResolver;
+
+    public function __construct(
+        $basePath,
+        VersionStrategyInterface $versionStrategy,
+        ContextInterface $context = null,
```

Comment:
I know we should not change the parent function arguments order, but in that case the lesser evil will be to place it as a last function argument, as it's optional.